pkg/workload/pgx_helpers.go

```diff
@@ -61,9 +61,6 @@ func NewMultiConnPool(
 		connsPerPool := distributeMax(connsPerURL[i], maxConnsPerPool)
 		for _, numConns := range connsPerPool {
 			connCfg, err := pgxpool.ParseConfig(urls[i])
-			// Disable the automatic prepared statement cache. We've seen a lot of
-			// churn in this cache since workloads create many of different queries.
-			connCfg.ConnConfig.BuildStatementCache = nil
```

Comment:
i vote that we should still keep this cache disabled, but curious to hear any thoughts on this. i don't feel too strongly, but my reasoning is just that it's not that obvious to a writer of a workload that this statement cache will cause all statements to be prepared. but perhaps that doesn't matter so much.